anti.md

```diff
@@ -46,6 +77,10 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There are some DSLs in this domain, like [RSS](http://www.whatisrss.com/). This
+appear to be somewhat widely supported, but I don't know anyone who uses it. It
+can do well with some things like blogs, but for other types of content it is
+not useful, and the website creator has to specifically support and publish it.
```

Comment:
There are some services like Pushbullet, which also just provide APIs that other people have to use, but that are in common usage.